src/test/java/org/jabref/logic/exporter/ExporterTest.java

```diff
@@ -50,40 +50,37 @@ public void setUp() {
         entries = Collections.emptyList();
     }
 
+    @Parameterized.Parameters(name = "{index}: {1}")
+    public static Collection<Object[]> exportFormats() {
+        Collection<Object[]> result = new ArrayList<>();
+
+        Map<String, TemplateExporter> customFormats = new HashMap<>();
+        LayoutFormatterPreferences layoutPreferences = mock(LayoutFormatterPreferences.class);
+        SavePreferences savePreferences = mock(SavePreferences.class);
+        ExporterFactory exporterFactory = ExporterFactory.create(customFormats, layoutPreferences, savePreferences);
+
+        for (Exporter format : exporterFactory.getExporters()) {
+            result.add(new Object[]{format, format.getDisplayName()});
+        }
+        return result;
+    }
+
     @Test
     public void testExportingEmptyDatabaseYieldsEmptyFile() throws Exception {
         File tmpFile = testFolder.newFile();
```

Comment:
Instead of calling `toPath` below, it would be better if you created a `Path` object here directly. That applies to the other tests as well.